backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -538,6 +484,7 @@ export class UserResolver {
         throw new Error('error saving user: ' + error)
       })
 
+      // why do we delete the code?
```

Comment:
yes why?

Reply:
In the login server this seems to be implemented differently - the old data contains multiple codes for each user